app/src/components/governance/PageProposals.vue

```diff
@@ -2,33 +2,59 @@
   <TmPage
     data-title="Proposals"
     :loading="$apollo.queries.proposals.loading && !loaded"
+    class="proposals"
   >
-    <div v-if="loaded" class="button-container">
-      <TmBtn
-        id="propose-btn"
-        value="Create Proposal"
-        type="secondary"
-        @click.native="onPropose"
-      />
-      <TmBtn
-        v-if="
-          connection.network === 'cosmos-hub-mainnet' ||
-          connection.network === 'cosmos-hub-testnet'
-        "
-        id="tutorial-btn"
-        class="tutorial-btn"
-        value="Want to learn how governance works?"
-        type="secondary"
-        @click.native="openTutorial()"
-      />
+    <div class="overview-header">
+      <div v-if="loaded" class="overview-top">
+        <h1>Governance Overview</h1>
+        <div>
+          <TmBtn
+            v-if="currentNetwork.network_type === `cosmos`"
+            id="propose-btn"
+            value="Create Proposal"
+            type="secondary"
+            @click.native="onPropose"
+          />
+          <TmBtn
+            v-if="currentNetwork.network_type === `cosmos`"
+            id="tutorial-btn"
+            class="tutorial-btn"
+            value="Want to learn how governance works?"
+            type="secondary"
+            @click.native="openTutorial()"
```

Comment:
let's replace the tutorials with just links to outside content by James

Reply:
another PR? Please, open an issue about this

Reply:
another PR !